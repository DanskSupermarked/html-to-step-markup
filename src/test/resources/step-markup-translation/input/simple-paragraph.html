<p> (TA) FÃ˜TEX.DK 1-EA</p>
