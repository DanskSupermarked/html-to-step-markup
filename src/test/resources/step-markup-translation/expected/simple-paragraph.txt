(TA) FÃ˜TEX.DK 1-EA
